ansible/files/runtimes.json

```diff
@@ -16,7 +29,7 @@
                 }
             },
             {
-                "kind": "nodejs:8",
+                "kind": "nodejs:18",
```

Comment:
Typo.

Reply:
Shoot... I had that change in the file for a test... missed to remove it.